invoke.yaml

```diff
@@ -0,0 +1,4 @@
+app: test_app
+repository: TEST_REPO
+run:
+  pty: True
```

Comment:
Is this new file related to the docker-compose change?

Reply:
It is needed to test any of the tasks that ask for the `app`. I added it  to the repo since it seemed convenient to have the ability to test tasks in the repo rather pulling it into an actual project.  But maybe that just means we should add real tests.

Reply:
I'm fine with it so long as it's not packaged so it can't end up in the projects using kubesae.

Reply:
Just to clarify... from my read of setup.py, nothing in the top level will be packaged, so this should be good, right? If so, I can merge this and release.

Reply:
Yes, from some testing, it looks like that is the case. I'm going to merge and deploy this so that I can use it with Pressweb.